x-pack/plugins/data_usage/server/services/autoops_api.ts

```diff
@@ -44,10 +44,11 @@ export class AutoOpsAPIService {
     }
 
     logger.debug(
```

Comment:
>[!NOTE]
> This `REDACTED` thing feels a bit useless, since if `autoopsConfig` is defined, then `api?.tls?` should also be fully defined since these are required, otherwise we should throw an exception.